src/lib/models/event-history/get-event-categorization.ts

```diff
@@ -4,72 +4,75 @@ import type {
   IterableEvent,
 } from '$lib/types/events';
 
+type Categories = typeof CATEGORIES;
+export type EventTypeCategory = Categories[keyof Categories];
+
+export const CATEGORIES = {
```

Comment:
Changed this to make it easer to use auto-complete, but I'm by no means attached to this format at all if there are objections 😅